charon/tests/ui/issue-92-nonpositive-variant-indices.rs

```diff
@@ -0,0 +1,13 @@
+//@ output=pretty-llbc
```

Comment:
My understanding is that this PR requires https://github.com/AeneasVerif/charon/pull/107 to be merged before?

Reply:
No, I just added a few test cases from open issues. As you can see, CI passes (as you can see, the match is weird)

Reply:
Indeed